.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: cpp
 sudo: required
 
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
```

Comment:
Reading http://clang.llvm.org/docs/CrossCompilation.html

Looks like it would be possible... allowing it to compile the other targets would be nice indeed, but it needs more work than what's done here.

> The Waf build system has the toolchain that should be used for the different targets but it is generating the name for the binary to be used.

You can see that at its core it's actually prefixing the triple to the name of the binary. It could just use the triple as an argument.